### PR TITLE
Implement functions

### PR DESCRIPTION
**Pull request by panferovi**

### CODE REVIEW

File: file/bytecode.h [hunk 1]
```diff
@@ -2,22 +2,55 @@
...
+            Method::Destroy(method);
+        }
+    }
+
+    using MethodBody = std::function<Body(StringTable &, const BytecodeMeta &)>;
+
+    void CreateMethod(const MethodBody &body, Signature signature, std::string_view method_name)
+    {
+        auto id = methods_.size();
+        auto *method = Method::Create(body(string_table_, meta_info_), std::move(signature), method_name);
```
Comment (ivan-egorov42) [inline file/bytecode.h:31]: `Method::Create` inside CreateMethod 💀

File: file/meta.h [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    {
+        auto method_iter = methods_.find(method_name);
+        if (method_iter == methods_.cend()) {
+            return INVALID_METHOD_ID;
+        }
+        return method_iter->second;
+    }
+
+private:
+    std::unordered_map<std::string_view, MethodId> methods_;
```
Comment (ivan-egorov42) [inline file/meta.h:32]: Mb `methods_id` or smth like that as we have `methods` inside `class Bytecode`?

File: file/string_table.h [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+#define FILE_STRING_TABLE_H_
+
+#include <vector>
+#include <string_view>
+
+#include "common/macros.h"
+
+namespace mipt_vm {
+
+class StringTable {
```
Comment (ivan-egorov42) [inline file/string_table.h:11]: Mb need constructor?

**Review by ivan-egorov42 — APPROVED** [review]

File: file/string_table.h (hunk 3 above)
Comment (panferovi) [reply file/string_table.h:11]: Mb ebalo off?

**Pull request merged**